doc/release/technotes/local.rst

```diff
@@ -78,3 +81,46 @@ as well as all memory referenced during the calls of ``A.this(5)``
 are located on the current locale. Otherwise an error is reported.
 Analogously, if ``on`` statement(s) are executed during these calls
 that attempt to execute on a different locale, an error is reported.
+
+
+The 'local on' Statement
+========================
+
+
+The ``local on`` construct in Chapel performs an on-statement on a sublocale
+within the current node. For example:
+
+.. code-block:: chapel
+
+  for i in 0..#here.getChildCount() {
+    local on here.getChild(i) {
+      writeln("On sublocale ", here);
+    }
+  }
+
+When the ``--local-checks`` flag is enabled, a runtime check will be inserted
+to confirm that the on-statement is performed within the same node.
+``--local-checks`` is enabled by default and can be disabled with
+``--no-local-checks``, ``--no-checks``, or ``--fast``. For example, the
+following code would produce a runtime error if the number of locales is
+greater than one:
+
```

Comment:
Yeah, and I knew what you meant, it's just my persnickety programmer side coming out.  :-)
Either of those solutions sounds fine.  I have a slight preference for the latter, but your choice rules.